src/cli/fleet.toit

```diff
@@ -122,28 +126,82 @@ class FleetFile:
           ui.abort "Fleet file $path has invalid format for 'pod' in group '$group-name'."
         PodReference.parse entry["pod"] --ui=ui
 
-    broker-config := default-broker-config
-
     broker-name := fleet-content.get "broker"
+    migrated-from-entry := fleet-content.get "migrated-from"
```

Comment:
Maybe this should be more like 'migrating-from' since it isn't finished yet?

Reply:
done.